File: FoodDelivery.Web/src/app/models/ownerRequest.ts
import { userProfile } from './userProfile';
import { roleRequestStatus } from './enums/roleRequestStatus';
import { Guid } from "guid-typescript";

export class ownerRequest
{
    id: Guid;
    userProfileId: Guid;
    createdDate: string;
    closedDate: string
    status: roleRequestStatus

    userProfile: userProfile;
}

File: FoodDelivery.DAL/EF/Entities/OwnerRequest.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace FoodDelivery.DAL.EF.Entities
{
	public class OwnerRequest
	{
		[Key]
		public Guid Id { get; set; }

		public Guid UserProfileId { get; set; }

		public int Status { get; set; }

		public DateTime CreatedDate { get; set; }

		public DateTime? ClosedDate { get; set; }

		public virtual UserProfile UserProfile { get; set; }
	}
}
